Dmitriy_Vysotskyy2.0/WebDriverFactory.cs

```diff
@@ -0,0 +1,22 @@
+using System.Data;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+
+namespace Dmitriy_Vysotskyy2._0;
+
+public class WebDriverFactory
+{
+    public static IWebDriver GetWebDriver(BrowserType browserType)
+    {
+        switch (browserType)
+        {
+            case BrowserType.Edge:
+                return new EdgeDriver();
+            case BrowserType.Chrome:
+                return new ChromeDriver();
+            default:
+                throw new Exception();
```

Comment:
Exception message is missing. Add some message which indicates that provided browsed type is not available